module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -554,6 +554,7 @@ public function getMyProfile($patron)
             'address1' => $address,
             'zip' => $zip,
             'city' => $city,
+            'birthdate' => $result['birthDate'] ?? '',
```

Comment:
Is the format of this guaranteed across all Sierra installations?

Reply:
Yes. See https://sandbox.iii.com/iii/sierra-api/swagger/index.html#!/patrons/Get_a_list_of_patrons_get_1. It's been stable across API versions.